### PR TITLE
load-test: fix failing tests due to role session timeout, clean up code

### PR DESCRIPTION
**Pull request by PettitWesley**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: buildspec_load_test.yml [hunk 1]
```diff
@@ -52,21 +52,13 @@ phases:
       - python -m venv venv
       - source venv/bin/activate
       - pip install -r ./load_tests/requirements.txt
-      # AWS configure sts temp credentials
```
Comment (PettitWesley) [inline buildspec_load_test.yml:55]: You may wonder, why did he remove this? Is this safe?

In load tests, everything runs via the python script, and as you can see in the other code changes, that code assumes the `LOAD_TEST_CFN_ROLE_ARN` role itself. This means we were previously unnecessarily chaining sessions (buildspec assumes the role, then the python uses that role to assume the same role).

File: load_tests/load_test.py [hunk 2]
```diff
@@ -224,7 +223,13 @@ def run_ecs_tests():
         # Validation input type banner
         print(f'\nTest {input_logger["name"]} to {OUTPUT_PLUGIN} in progress...')
 
+    # wait for tasks and validate
+    for input_logger in INPUT_LOGGERS:
```
Comment (PettitWesley) [inline load_tests/load_test.py:227]: This change should speed up the tests slightly. If you read the lines above and below here, we first start the tasks for each input (tcp vs stdout) and then validate... but each of our tests uses a unique resource (log stream, firehose, kinesis, etc) for each test... so there's no reason why we can't start all of them first and then loop over them and check for results.

**Review by weijianz-opc — APPROVED** [review]
No concern.

File: load_tests/load_test.py [hunk 3]
```diff
@@ -200,7 +200,6 @@ def run_ecs_tests():
         session = get_sts_boto_session()
 
         client = session.client('ecs')
-        waiter = client.get_waiter('tasks_stopped')
         
         processes = []
```
Comment (DrewZhang13) [inline load_tests/load_test.py:205]: Do we till need this for loop after logic change? I don't understand what's the work for this for-loop.
At least should we remove `processes = []`  out of the for loop?

Comment (PettitWesley) [reply load_tests/load_test.py:205]: oh you're right, thanks, processes needs to move to below.

Comment (PettitWesley) [reply load_tests/load_test.py:205]: https://github.com/aws/aws-for-fluent-bit/pull/677

Comment (PettitWesley) [reply load_tests/load_test.py:205]: This for loop runs the tasks, then 2nd one validates.

**Pull request merged**